webpack.config.js

```diff
@@ -28,36 +22,48 @@ module.exports = {
         }
       }
     ]
+  }
```

Comment:
I would remove `module.rules.exclude`. I think the `include` already does the whitelisting, so there's no need to do the blacklisting as well.

Reply:
It does, i misinterpreted the rule the first time i dabbled into babel-loader. The exclude is only **for excluding subfolders inside the include**, everything else is auto excluded. In our library we do not need to do that, so i removed it before merging. Good catch! :+1: